src/schemathesis/cli/output/junitxml.py

```diff
@@ -0,0 +1,46 @@
+import platform
```

Comment:
Could you, please, move it out from `output` package to `cli`? these ones are only for stdout